cmake/kokkos_functions.cmake

```diff
@@ -954,6 +954,86 @@ int main()
   set(${_VAR} ${_RET} CACHE STRING "CXX compiler supports building CUDA")
 endfunction()
 
+# this function checks if the compiler supports specific flags. No-op for CMake < 3.19:
+#
+#       SILENT    --> check without printing the check (default=ON)
```

Comment:
How do we set this variable?

Reply:
good catch, I forgot to add it to the `parse_arguments`